buildpacks-18/end-of-life-buildpack/bin/build

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ENV_DIR="${2}/env"
+ALLOW_BUILD="$(cat "${ENV_DIR}/ALLOW_INSECURE_HEROKU_18_BUILDER" 2> /dev/null || true)"
+
+function echo_stderr() {
+  local ansi_red="\033[1;31m"
+  local ansi_reset="\033[0m"
+  echo -e "\n${ansi_red}${1}${ansi_reset}\n" >&2
+}
+
+read -r -d '' EOL_MESSAGE <<'EOF' || true
+This builder image (heroku/buildpacks:18) is based upon the Heroku-18
+stack, which has been end-of-life since April 30th, 2023:
+https://devcenter.heroku.com/changelog-items/2583
+
+The underlying Ubuntu 18.04 OS is no longer receiving security updates,
+and so apps still using it could contain security vulnerabilities.
```

Comment:
Sidenote: I can't make a suggested change across multiple lines, unfortunately. 

Enhancement: This wording is a little awkward. I think we could replace `, and so` with a full stop. Like:

```
The underlying Ubuntu 18.04 OS is no longer receiving security updates. Apps still using it could contain security vulnerabilities.
```

I think that the wording "contain security vulnerabilities" makes it sound like the vulnerability is coming from within the app. Perhaps:

```
The underlying Ubuntu 18.04 OS is no longer receiving security updates. Apps still using this OS could be vulnerable.
```

I'm not in love with my version, either. It's a difficult thing to say succinctly and correctly.

Reply:
Thank you, I like your wording - I've updated it in 3f65199